src/build.rs

```diff
@@ -64,7 +64,15 @@ pub struct RepoBuilder<'cb> {
 pub type RemoteCreate<'cb> =
     dyn for<'a> FnMut(&'a Repository, &str, &str) -> Result<Remote<'a>, Error> + 'cb;
 
-/// A builder struct for git tree updates, for use with `git_tree_create_updated`.
+/// A builder struct for git tree updates.
+///
+/// Paths passed to `remove` and `upsert` can be multi-component paths, ie they
+/// may contain slashes.
+///
+/// This is the more-cooked tree update facility.  There is also [`TreeBuilder`]
```

Comment:
```suggestion
/// This is a higher-level tree update facility.  There is also [`TreeBuilder`]
```

(Parallel construction with "lower-level".)